source/includes/administration/_environments.md

```diff
@@ -22,26 +22,34 @@ curl "https://cloudmc_endpoint/api/v1/environments" \
     "id": "1ee5cd43-8395-4cd5-a20f-0f1e83b7f8bd",
     "name": "cheyenne_mountain",
     "description": "Environment for base at Cheyenne Mountain",
+    "metadata": {
+      "mode": "project"
+    },
+    "defaultRole": {
+      "creationDate": "2020-01-02T16:04:27.000Z",
+      "version": 1,
+      "isSystem": false,
+      "isDefault": false,
+      "deleted": false,
+      "name": "Owner",
+      "alias": "owner",
+      "id": "b7c48102-350e-40b9-9a3e-b7a0b3cfa535",
+      "isFixed": true,
+      "defaultScope": "ENV"
```

Comment:
I don't think we should include `version` and `deleted` attributes. If I'm not mistaken, `alias` is a legacy field that is deprecated. We should also order the fields in a logical fashion for the example to make it easier to understand.

```suggestion
      "id": "b7c48102-350e-40b9-9a3e-b7a0b3cfa535",
      "name": "Owner",
      "isSystem": false,
      "isDefault": false,
      "isFixed": true,
      "defaultScope": "ENV",
      "creationDate": "2020-01-02T16:04:27.000Z"
```